src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let res = 0;
+  const a = days * 40;
+
+  if (days < 3) {
+    return a;
+  }
+
+  if (days < 7) {
+    res = a - 20;
+
+    return res;
+  }
+
+  if (days >= 7) {
+    res = a - 50;
+
+    return res;
+  }
```

Comment:
Looks like you haven't properly read the [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md), cause your solution doesn't meet 1st, 2nd and 4th criterias